### PR TITLE
broken-links

### PR DESCRIPTION
**Pull request by hajunho**
Essential links that help newbies have been broken.

## Change Type
**Select *ONE*:**
- [x] Documentation

- [ ] Stability bug fix

- [ ] Other

- [ ] Other - special case

## Testing Changes
**Select *ANY* that apply:**
- [ ] New Tests

- [ ] Existing Tests

- [ ] Test Framework

- [ ] CI System

- [ ] Other

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [x] Documentation Additions

### CODE REVIEW

**Review by heifner — COMMENTED** [review]
All PRs should target `develop` and/or one of the release branches. e.g. `release/2.2.x`.

File: README.md [hunk 1]
```diff
@@ -121,13 +121,13 @@ To uninstall the EOSIO built/installed binaries and dependencies, run:
...
-    - [Troubleshooting](http://eosio.github.io/eos/latest/nodeos/troubleshooting/index)
-1. [Cleos](http://eosio.github.io/eos/latest/cleos/)
-1. [Keosd](http://eosio.github.io/eos/latest/keosd/)
+1. [Nodeos](https://developers.eos.io/manuals/eos/latest/nodeos/index)
+    - [Usage](https://developers.eos.io/manuals/eos/latest/nodeos/usage/index)
+    - [Replays](https://developers.eos.io/manuals/eos/latest/nodeos/replays/index)
+    - [Chain API Reference](https://developers.eos.io/manuals/eos/latest/nodeos/plugins/chain_api_plugin/api-reference/index)
+    - [Troubleshooting](https://developers.eos.io/manuals/eos/latest/nodeos/troubleshooting/index)
+1. [Cleos](https://developers.eos.io/manuals/eos/latest/cleos/index)
+1. [Keosd](https://developers.eos.io/manuals/eos/latest/keosd/index)
```
Comment (heifner) [inline README.md:131]: Thanks @hajunho. @lparisc can we get these fixed please.

Comment (lparisc) [reply README.md:131]: @heifner I had fixed these broken links on `develop` and `release/2.2.x` branches. I think @hajunho intended a hotfix on `master` as well - although his branch was not based off `master`, hence the other commits. I will cherry-pick his commit to an internal branch then fix `release/2.1.x` and (hotfix) `master` - I haven't had much success with external PRs lately as far passing all buildkite tests.

**Review by lparisc — CHANGES_REQUESTED** [review]

**Comment by lparisc** [comment]
Closing this PR since changes combined with PR #10544 and #10546.

**Pull request closed without merging**